src/components/token-table/__stories__/token-table.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import TokenTable from '../token-table';
+
+const meta: Meta<typeof TokenTable> = {
+  component: TokenTable,
+  title: 'Components/TokenTable',
+};
+export default meta;
+
+type Story = StoryObj<typeof TokenTable>;
+
+export const Overview: Story = {
+  args: {
+    theadData: ['Column A', 'Column B'],
```

Comment:
```suggestion
    theadData: ['Token', 'Value'],
```

We are just mostly using it to display smth like this
![image](https://github.com/mondaycom/vibe-storybook-components/assets/104433616/b087d4e5-8d7e-4dd0-a302-9179b456c042)